src/rpc/masternode.cpp

```diff
@@ -587,7 +588,16 @@ static UniValue masternodelist(const JSONRPCRequest& request)
         return (int)pindex->nTime;
     };
 
+    bool showAllBannedMns = strMode != "recent";
+    int tipHeight = WITH_LOCK(cs_main, return ::ChainActive().Tip()->nHeight;);
+    constexpr int monthBlockLength = 24 * 24 * 30;
```

Comment:
nit: s/Length/Count/ maybe?

also, could probably use `Params().GetConsensus().nSuperblockCycle` instead - would make it much easier to write tests for this mode cause regtest cycles are much smaller (10 blocks). Though I'm not sure if service desk guys are going to be happy about it cause testnet/devnet cycles are also pretty small (24 blocks).